### ISSUE TITLE
'/bin/sh: 1: dotnet: not found' on docker 10.5.0

### ISSUE THREAD

**Issue by OlavStornes** [jellyfin/jellyfin#2545]
**Describe the bug**

After updating jellyfin, i couldn't access my instance, as it didn't initialize correctly. 

**System (please complete the following information):**
 - OS: Ubuntu
 - Virtualization: Docker
 - Clients: Not relevant
 - Browser: Not relevant
 - Jellyfin Version: 10.5.0, 
 - Playback: Not relevant
 - Installed Plugins: Playback reporting, TMdb box sets
 - Reverse Proxy: traefik
 - Base URL: no
 - Networking: Bridge
 - Storage: Config on local, media mounted on host over samba

**To Reproduce**

1. Start from a previous jellyfin-instance on docker. (version 10.4.5 in my case)
2. Update to latest/10.5.0
3. See output of the newly created container

**Expected behavior**

Being able to access my instance of jellyfin

**Logs**

`/bin/sh: 1: dotnet: not found` repeats itself

**Screenshots**

Not applicable

**Additional context**

[Other people on reddit experience the same as well](https://www.reddit.com/r/jellyfin/comments/ffm357/jellyfin_1050_released/fjzivnz/)

**Comment by Artiume** [comment]
I'm guessing you're using portainer. 

https://github.com/portainer/portainer/issues/3604

Can you go here and describe how you update the container

**Comment by OlavStornes** [comment]
Thanks! Yes, i did indeed use portainer for the update. After inspecting the docker image, the issue on portainer and the linked post on reddit, the issue seems to be that portainer is not updating it's entrypoint if the image is changed. 


It seems to be two ways to remedy this:
- Remove the old container and redeploy (compared to editing/updating directly in portainer)
- Ensure that the entrypoint is: `./jellyfin/jellyfin --datadir /config --cachedir /cache --ffmpeg /usr/local/bin/ffmpeg`

I removed the old container and used docker-compose to create the new, which made jellyfin work as intended again.

Closing this since it isn't directly related to jellyfin.
